src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/CSharp/Microsoft.Extensions.AI.Evaluation.Reporting.csproj

```diff
@@ -26,7 +26,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\Microsoft.Extensions.AI\Microsoft.Extensions.AI.csproj" />
+    <PackageReference Include="Microsoft.Extensions.AI" VersionOverride="9.3.0-preview.1.25161.3" />
+    <!-- <ProjectReference Include="..\..\Microsoft.Extensions.AI\Microsoft.Extensions.AI.csproj" /> -->
```

Comment:
@joperezr We were not sure if swapping project refs with package refs is a good idea in this release branch. It reflects reality of what we need to do for the current release - but in case we decide to do another one-off release from this branch, we'll need to remember to update this before that...

Reply:
Yeah, this is not ideal but I understand why you have to do it. As requested in a previous comment, can you please add a comment explaining this change so we remember to revert it?

Reply:
Comments added to each reference regarding removing the version override for new builds on this branch.